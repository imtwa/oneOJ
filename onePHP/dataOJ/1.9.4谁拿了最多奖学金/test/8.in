90
LOKPI 94 92 Y Y 1
GPIMF 96 99 Y Y 3
OHLPJ 81 93 Y Y 4
JGPHF 95 82 N N 4
IMPNF 92 80 N N 0
JKHKN 86 92 N N 1
JIJMM 95 87 Y N 7
MNPPF 96 93 Y Y 9
OLPGG 87 80 N Y 0
HOKKG 94 89 N N 8
GIPIO 99 85 N Y 6
HJPNM 84 98 N N 9
HNFII 83 86 N N 5
HFMKP 96 91 N N 2
HOOGN 89 99 Y N 1
IMKOH 81 98 N N 8
FGJKG 83 82 N N 0
OKOMJ 94 94 N N 2
JHMGI 90 91 Y N 7
NGFGK 98 89 N Y 7
MOOPL 87 90 Y Y 3
PFPJK 92 93 N Y 4
LGNGJ 80 85 N N 0
LLKNM 85 89 N Y 3
NGNNF 94 92 N Y 1
NHJOH 98 91 N N 1
LPNIL 95 86 Y Y 7
KFOKH 84 90 N Y 7
JGLFO 92 81 Y N 9
GOPMJ 86 88 Y Y 3
PONOG 93 91 Y N 2
LJJMK 95 97 N Y 4
HOOMP 82 82 N Y 1
JNGFF 90 91 Y N 4
ONPHG 90 96 Y Y 5
LONOI 100 96 Y Y 3
JNPFN 91 91 N N 5
FPPGP 90 87 Y Y 5
OFILI 91 86 Y Y 4
JGJFL 89 89 N Y 1
LOFMF 84 99 N Y 3
OLMLP 88 87 Y Y 4
MLNHM 94 87 N Y 7
GLMFF 99 87 Y Y 8
KIPFN 90 84 Y N 3
ILHIH 89 99 Y N 4
OPGJP 88 81 N N 3
JMKOH 83 89 Y Y 3
JPHPM 88 80 Y N 3
NKKFM 93 92 Y Y 4
GNPOL 86 93 N N 4
OOFJG 80 91 N N 1
FNOJI 89 98 Y N 6
KKGFG 85 100 N Y 9
OHJGJ 80 97 Y Y 2
NOGOI 93 94 Y N 2
KHOGI 95 89 N N 3
JPJKL 80 95 Y Y 2
LNKJK 93 90 N Y 0
PLLJJ 96 88 Y Y 6
OGKLK 98 86 N N 8
OPNGH 98 90 N Y 9
LNOOO 92 97 Y N 9
GKIFH 96 96 N N 1
KOOIM 89 91 Y Y 3
HGKFK 94 91 N Y 3
IJHMN 96 95 Y N 6
FKMKK 80 89 N N 5
NOGMI 84 80 Y N 3
HKOHP 85 86 N N 1
ILGJM 86 89 N N 6
LLPGN 100 81 N Y 7
MMMFN 99 82 Y Y 5
HMIPM 86 99 Y N 9
OPJLO 93 89 Y Y 0
MFIML 99 95 Y Y 6
FJFOO 82 82 N Y 1
PNOJN 95 88 Y Y 4
KHKGP 82 90 Y Y 0
KHIPK 92 80 N Y 1
MLJKN 97 100 Y N 1
PMMNK 98 83 N Y 9
JLPIG 95 93 Y N 6
PHOIH 93 99 N Y 3
LMGFF 83 96 N N 0
MFFML 98 88 N Y 0
HFLFL 94 94 Y N 6
MKLLM 87 80 Y Y 1
FINFF 100 81 Y Y 1
JKGIG 91 81 Y Y 0
