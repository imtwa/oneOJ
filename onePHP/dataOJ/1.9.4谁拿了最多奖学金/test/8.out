LOKPI
15850
1033700
